en/micro-integrator/docs/develop/add-new-remote-server.md

```diff
@@ -0,0 +1,45 @@
+# Adding a remote Micro Integrator or ESB Server
+
+The following instructions can be used to run your artifacts in a remote Micro Integrator or an ESB Server
+
+## Configure and run artifacts in a remote Micro Integrator or ESB Server
+
+1.  Open the <b>Getting Started</b> view and click <b>Add Server</b> to open the <b>New Server</b> dialog.
+    
+    <img src="../../assets/img/testing-integrations/is_getting_started_page.png" width="700">
+
+2.  In the <b>New Server</b> dialog that opens, expand the WSO2 folder and select the version of your server.
+
+    <img src="../../assets/img/testing-integrations/new-server-choose-type.png" width="500">
+
+3.  Click <b>Next</b>. In the CARBON_HOME field, provide the path to your product's home directory and then click <b>Next</b> again.
+
+4.  Review the default port details for your server and click <b>Next</b>.
+
+    - If you selected a <b>Micro Integrator</b> server in the previous step, enter the port details required for a Micro Integrator:
+    
+    <img src="../../assets/img/testing-integrations/new-server-mi-port-settings.png" width="500">
+
+    - If you selected an <b>Enterprise Integrator</b> server in the previous step, enter the port details required for an Enterprise Integrator:
+    
+    <img src="../../assets/img/testing-integrations/new-server-ei-port-settings.png" width="500">
+
+    !!! Note
+        If you are already running another server on these ports, give unused ports. See [Default ports](../../setup/changing_default_ports) of WSO2 Micro Integrator for more information.
+
+5.  To deploy the C-App project to your server, select the composite application from the list, click <b>Add</b> to move it into the configured list, and then click <b>Finish</b>.
+    <img src="../../assets/img/testing-integrations/new-server-add-capps.png" width="500">
+
+6.  On the <b>Servers</b> tab, note that the server is currently stopped. Click the <img src="../../assets/img/common/play-head-icon.png" width="20"> icon on the tool bar. If prompted to save changes to any of the artifact files you created earlier, click <b>Yes</b>.
+
+    <img src="../../assets/img/testing-integrations/new-server-start.png" width="500">
+
+7.  You can also deploy/redeploy or remove C-Apps from a running server:
+    
+    - To deploy/remove C-Apps, right-click the server, click <b>Add and Remove</b> and follow the instructions on the wizard.
+
+    <img src="../../assets/img/testing-integrations/new-server-add-remove-capps.png" width="500">
+
+    - If you want to redeploy a C-App after modifying the included artifacts, select the already deployed C-App, right-click and click <b>Redeploy</b>.
+    Hot deployment is enabled in the Micro Integrator by default. This allows you to redeploy artifacts without restarting the server.
+    - If you have used an older ESB Server, select the server, right-click and click <b>Restart</b> to redeploy the updated artifacts.
```

Comment:
instead of older ESB, we need to use some other term. @nilminiwso2

Reply:
We needn't talk about ESB servers in these docs..
We are adding a remote Micro Integrator...
@Nirothipan

Reply:
seems okay, then shall we limit only to remote MI @SanojPunchihewa